site/content/docs/5.3/migration.md

```diff
@@ -9,6 +9,15 @@ aliases:
 toc: true
 ---
 
+## From Boosted
+
+* New `$enable-bootstrap-specificities` option set to `true` by default.
+  * Must be a global migration note as it will impact not only the opacity utilities
+* New opacity utilities: `.opacity-transparent`, etc.
+* Deprecated opacity utilities: `.opacity-0`, etc.
```

Comment:
Just to be noted: you can't use the vocabulary "deprecated" since these utilities will never be removed

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2573/commits/625c76148b1fa68ae0ed089e8ef55f5065f96999 for this temp version really quickly written (just to not forget it)